### DISCUSSION TITLE
Authentication with Active Directory on premise (ADFS)

### DISCUSSION THREAD

**Discussion by Ach1m9** [nextauthjs/next-auth#4828] [Help]
Hello to all,

in the feature list of next-auth we can read about "Supports stateless authentication with any backend (Active Directory, LDAP, etc)", but I didn't see any documentation about a special provider or code snippets to implement this in my next.js App.

Is there somebody who implemented Active Directory/ADFS? I hope :)

**Comment by AnishK97** [comment]
Hi did you find any documentation related to ADFS?

**Comment by abonnell** [comment]
Bumping on this, were you able to find any resources/examples or did you implement your own custom provider and would be willing to share?
